wgpu-core/Cargo.toml

```diff
@@ -19,6 +19,7 @@ default = []
 trace = ["ron", "serde", "wgt/trace"]
 replay = ["serde", "wgt/replay"]
 metal-auto-capture = ["gfx-backend-metal/auto-capture"]
+x11 = ["gfx-backend-vulkan/x11"]
```

Comment:
Let's remove this, and also remove `metal-auto-capture` since we are at it
The user can always just add a dependency on `gfx-backend-vulkan` directly, enabling features they need.

Reply:
That sounds awesome!